scipy-1.0/paper.tex

```diff
@@ -239,6 +239,24 @@ \subsection*{Data structures}
 \input{ckdtree}
 \textbf{Sparse matrices}
 
```

Comment:
Would be good to start with a sentence like: "`scipy.sparse` offers seven sparse matrix data structures. The most important ones are CSR and CSC, these offer fast row or column access and fast matrix-vector multiplication, and are used heavily in scikit-learn. <then to the performance improvement>"